interface/HHAnalyzer.h

```diff
@@ -61,25 +61,19 @@ class HHAnalyzer: public Framework::Analyzer {
                 }
             }
         }
+        virtual void endJob(MetadataManager&) override;
 
         // leptons and dileptons stuff
         BRANCH(electrons, std::vector<unsigned int>);
         BRANCH(muons, std::vector<unsigned int>);
         BRANCH(leptons, std::vector<HH::Lepton>);
-        BRANCH(ll, std::vector<HH::Dilepton>);
         BRANCH(met, std::vector<HH::Met>);
-        BRANCH(llmet, std::vector<HH::DileptonMet>);
         BRANCH(jets, std::vector<HH::Jet>);
-        BRANCH(jj, std::vector<HH::Dijet>);
-        BRANCH(llmetjj, std::vector<HH::DileptonMetDijet>);
+        std::vector<HH::Dilepton> ll;
+        std::vector<HH::DileptonMet> llmet;
+        std::vector<HH::Dijet> jj;
+        std::vector<HH::DileptonMetDijet> llmetjj;
```

Comment:
I only see a clear for llmet and not for the others. Were there already there? Otherwise you need them since there are no longer branches